app/javascript/app/ssn-field.js

```diff
@@ -42,7 +42,8 @@ function formatSSNFieldAndLimitLength() {
         }
         const didFormat = input.value !== value;
         if (didFormat) {
-          input.checkValidity();
+          const isValid = input.checkValidity();
+          input.setAttribute('aria-invaid', !isValid);
```

Comment:
Typo:

```suggestion
          input.setAttribute('aria-invalid', !isValid);
```

Reply:
:crying-cowboy-sunglasses: thanks for the catch... unsure how my tests passed